src/quemb/molbe/autofrag.py

```diff
@@ -143,11 +258,33 @@ def remove_nonnunique_frags(self, natm: int) -> None:
                     del self.center[bdx]
                     del self.fsites[bdx]
                     del self.fs[bdx]
-                    del self.center_atoms[bdx]
-                    del self.fragment_atoms[bdx]
+                    del self.center_atom[bdx]
+                    del self.Frag_atom[bdx]
 
         return None
 
+    def to_FragPart(self, mol: Mole, be_type: str, frozen_core: bool) -> FragPart:
+        MISSING = []  # type: ignore[var-annotated]
+        return FragPart(
+            mol=mol,
+            frag_type="graphgen",
+            be_type=be_type,
+            edge_sites=self.edge_sites,  # type: ignore[arg-type]
+            edge_idx=MISSING,
+            center_idx=MISSING,
+            centerf_idx=self.centerf_idx,  # type: ignore[arg-type]
+            fsites=self.fsites,  # type: ignore[arg-type]
+            center=self.center,  # type: ignore[arg-type]
+            ebe_weight=self.ebe_weight,  # type: ignore[arg-type]
+            Frag_atom=self.Frag_atom,  # type: ignore[arg-type]
+            center_atom=self.center_atom,  # type: ignore[arg-type]
+            hlist_atom=MISSING,
+            add_center_atom=MISSING,
+            frozen_core=frozen_core,
+            iao_valence_basis=None,
+            iao_valence_only=False,
+        )
+
 
```

Comment:
For the `MISSING` items, is the plan to add these in a later PR or w/ #112?

Reply:
I forward this question to @ShaunWeatherly .

Note that the components are not missing because of this PR, but that the refactoring just makes it explicit.